autobuild/windows/autobuild_windowsinstaller_1_prepare.ps1

```diff
@@ -20,13 +22,26 @@ param(
 echo "Install Qt..."
 # Install Qt
 pip install aqtinstall
+if ( !$? )
```

Comment:
I had really hoped for `$ErrorActionPreference` to cover external command failures as well, but sadly it doesn't. [There doesn't seem to be an equivalent to bash's `set -eu`](https://stackoverflow.com/questions/60266883/) either. So, manual error checking is all we're left with.

I considered adding a function to do that to avoid code duplication (similar to `deploy_windows.ps1`'s Invoke-Native-Command), but I figured it's not worth it given the low number of repetitions within a single script.